README.md

```diff
@@ -23,6 +23,8 @@ Integration
 The checker uses the Symfony Console component; so, you can easily integrate
 the checker into your own project:
 
+ * by requiring it with Composer: `composer require ensiolabs/security-checker --dev`: https://symfony.com/doc/current/setup.html#checking-for-security-vulnerabilities
```

Comment:
I would not recommend it as your project deps can interfere with the checker ones.

Reply:
But it's in Symfony's own docs? Does it need to be removed from those docs then if you don't recommend it?

Reply:
It needs to be removed everywhere. It's the default value, so there is no need to be explicit here.

Reply:
Okay I'll send in a PR to remove it from the Symfony docs 👍